src/main/java/com/thevoxelbox/voxelsniper/brush/type/canyon/CanyonBrush.java

```diff
@@ -18,12 +18,12 @@
     @Override
     public void handleCommand(String[] parameters, Snipe snipe) {
         SnipeMessenger messenger = snipe.createMessenger();
-        String firstParameter = parameters[1];
+        String firstParameter = parameters[0];
         if (firstParameter.equalsIgnoreCase("info")) {
             messenger.sendMessage(ChatColor.GREEN + "y[number] to set the Level to which the land will be shifted down");
         }
         if (!firstParameter.isEmpty() && firstParameter.charAt(0) == 'y') {
-            int y = Integer.parseInt(firstParameter.replace("y", ""));
+            int y = Integer.parseInt(firstParameter.substring(1));
```

Comment:
See FlatOceanBrush comment and maybe leave a note if y is outside of the hardcoded range.